### PR TITLE
Krav Maga Code Update

### PR DESCRIPTION
**Pull request by PKPenguin321**
## THIS PR NO LONGER GIVES SEC KRAV MAGA

**See https://github.com/tgstation/-tg-station/pull/16843 for more info on Krav Maga**
- Gives sec an advantage over the crew that isn't some lameo baton
- Can be stolen since it's gloves and not an implant, opens it up to traitorous uses
- Krav is pretty much the weakest martial art we have so it's not OMG SLEEPIGN CARP FOR SEC REEE
- Traitor item version coming in a different PR (atomized features)

^ignore all of that stuff this doesnt give sec krav any more

Actual code changes in this PR:
- Commits sprites that I accidentally left out to actions.dmi
- Reduces leg sweep stun to Weaken(2)
- Creates and gives to sec a subtype of krav maga gloves that are very obvious, repaths the gloves to be children of black gloves
- Makes the implant toggle your krav maga on and off instead of just being one-use
- Adds myself to admins.txt
- Adds sprites for Krav Maga gloves, both ongrounds and onmobs

krav maga gloves

### CODE REVIEW

**Comment by Core0verload** [comment]
Aaaand in the downvotes hell you go.

**Comment by PKPenguin321** [comment]
dank feedback bro

**Comment by Supermichael777** [comment]
AND SO THE PAIN TRAIN OF ANTI SEC GRUDGECODE GETS MORE FUEL.

steelrifle all over again

**Comment by PKPenguin321** [comment]
ehhhh not really, steelrifles were basically instakill ggnore le loyalty implant :^^^)

**Comment by Supermichael777** [comment]
steel-rifles never got round-start sec pins because even steel saw how shitty that would be.
the armory remapping just made everyone butt-hurt.

**Comment by octareenroon91** [comment]
Are they still heatproof?

**Comment by Zilenan91** [comment]
What are the Krav Maga moves again? Also yeah the gloves need to be heatproof if you can make them.

**Comment by PKPenguin321** [comment]
question: should i give this to roundstart nuke ops too

**Comment by Iamgoofball** [comment]
get in coderbus

**Comment by Core0verload** [comment]
Why can't sec players just git gud? Seriously, the amount of powercreep starts to worry me.

**Comment by PKPenguin321** [comment]
i'm not a sec player i'm like borderline antisec, i just think this is hella cool

**Comment by Core0verload** [comment]
I remember a huge argument against stungloves was "OP UNDISARMABLE STUN NERF PLZ". And this resulted in stungloves being removed. And this PR is stungloves all over again, except now they are sec roundstarts.

What can possibly go wrong?

**Comment by Zilenan91** [comment]
This isn't stungloves though

**Comment by PKPenguin321** [comment]
stungloves could be carried by literally anybody with pretty much no tell, plus they had a full stunbaton level stun

i will make some tweaks to krav maga and the gloves to reduce the sekrit stun potential though

**Comment by Core0verload** [comment]
You, four days ago:
>balance tag
on an admin only feature

So admin-only, much balance.

**Comment by Bawhoppen** [comment]
pkp I will only allow this if the gloves are a good sprite
pic

**Comment by PKPenguin321** [comment]
gloves are normal sprite give better sprite i add it

@Core0verload at the time it most definitely was admin only

**Comment by Iamgoofball** [comment]
@Bawhoppen they're just default gloves right now because i didn't have any

@PKPenguin321 if you want you could use the Hallway Brawler glove sprites

**Comment by PKPenguin321** [comment]
that is smart

**Comment by KorPhaeron** [comment]
I can't decide if security running around punching everyone to death is funny or really stupid

**Comment by Zilenan91** [comment]
They have this on Hippie because of the tiding since they're on the hub. They might not need it here but it does seem pretty cool.

**Comment by Fox-McCloud** [comment]
>should i give this to roundstart nuke ops too

Absolutely

**Comment by Bawhoppen** [comment]
don't give people unneeded or unwanted things, ops do not need karate gloves and it does not fit their theme

**Comment by PKPenguin321** [comment]
@Bawhoppen krav maga is less "SUPER MEGA KUNG FU ACTION FIGHT MOVIE" and more "basic police combat training" (i *may* have overhyped the kung fu aspect in the irc)

**Comment by optimumtact** [comment]
It's time to stop

**Comment by KorPhaeron** [comment]
@Cheridan still hasn't recieved 200 dollars, so I  believe stun gloves of any sort are still banned

**Comment by PKPenguin321** [comment]
Don't meme me

**Comment by PKPenguin321** [comment]
Conflcits should be fixed

**Comment by optimumtact** [comment]
stop this madman

File: code/game/objects/items/weapons/implants/implant_krav_maga.dm [hunk 1]
```diff
@@ -19,8 +19,10 @@
 	var/mob/living/carbon/human/H = imp_in
 	if(!ishuman(H))
 		return
-	style.teach(H)
-	qdel(src)
+	if(istype(H.martial_art, /datum/martial_art/krav_maga))
+		style.remove(H)
+	else
+		style.teach(H,1)
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/implants/implant_krav_maga.dm]: Remove Krav Maga if they have it?

File: config/admins.txt [hunk 2]
```diff
@@ -112,3 +112,4 @@ Lumipharon = Game Master
 bgobandit = Game Master
 coiax = Game Master
 RandomMarine = Game Master
+PKPenguin321 = Game Master
```
Comment (KazeEspada) [inline config/admins.txt]: Stealth nerf

File: code/game/objects/items/weapons/implants/implant_krav_maga.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/implants/implant_krav_maga.dm]: yeah, so you can toggle it

File: config/admins.txt (hunk 2 above)
Comment (PKPenguin321) [inline config/admins.txt]: stop memeing me reeeeeeeeeeeeeee

**Comment by Nanotransistor** [comment]
Lets give the guys that already have armor, ranged AoE stuns, and disabler smgs a weapon that cannot be disarmed and doesnt need to be equiped to be used (sec will always have these gloves on).


This kind of stuff would only be ok if traitor sec was enabled to balance it out.

**Comment by PKPenguin321** [comment]
ty for real feedback
i am considering just making it a traitor/nuke op only item now but i'd still like player input on this PR
if anybody could run a poll ingame (cough @KorPhaeron) that would be super cool of you

**Comment by PKPenguin321** [comment]
yo yo yo check out the [forum thread/poll](https://tgstation13.org/phpBB/viewtopic.php?f=10&t=6466) right now and give a vote or something

right now it's looking like this will be given to only the warden instead of the entire sec team so dont get your jimmies too rustled

**Comment by Zilenan91** [comment]
It's tied with letting all sec officers have it. I see zero reason to not add it to everyone, it's not that powerful.

**Comment by Fox-McCloud** [comment]
Should the detective and lawyer? (lol) have this?

**Comment by Zilenan91** [comment]
Lawyer isn't sec, and the detective shouldn't have it either since he shouldn't be security in the first place.

**Comment by PKPenguin321** [comment]
nah detective gets a pew pew gun he doesnt need krav on top of it, plus he's only like borderline sec anyways
lawyers are definitely no but it's a funny thought

**Comment by demonfiren** [comment]
Lawyers only need neuralysers, nothing else.

**Comment by Fox-McCloud** [comment]
Yeah, I know, it was mostly a joke =p

**Comment by Fox-McCloud** [comment]
Still think nukies should have a box of these.

Also, should there be a box of these in the armory for people who change jobs to sec? That's largely what the tasers in the armory are for.

**Comment by Bawhoppen** [comment]
nukeops should not have these

**Comment by Iamgoofball** [comment]
Nuke ops are gonna get wrasslin instead

**Comment by Zilenan91** [comment]
Yeah nuke ops shouldn't have these.

**Comment by PKPenguin321** [comment]
>put up forum poll to get feedback that i will react to in a day or two
>a day later the forum dies

welp sit tight folks we're gonna have to wait on that to come back before this PR will progress anywhere

**Comment by Iamgoofball** [comment]
I was actually reading this thread when forums ded, it was 14(?) votes for add to all of sec leading.

**Comment by PKPenguin321** [comment]
okay new plan, i'm remodeling this PR into JUST the code upgrades and leaving the addition to sec as a separate PR

**Comment by phil235** [comment]
You can make the gloves temperature proof by just adding
```
 	cold_protection = HANDS
 	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
 	heat_protection = HANDS
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	burn_state = FIRE_PROOF
```

no need to change the kravmaga gloves's typepath to gloves/color/black.

File: code/modules/clothing/gloves/color.dm [hunk 3]
```diff
@@ -45,6 +46,9 @@
 
 /obj/item/clothing/gloves/color/black/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
 	if(istype(W, /obj/item/weapon/wirecutters))
+		if(!can_be_cut)
+			..()
```
Comment (phil235) [inline code/modules/clothing/gloves/color.dm]: the `..()` isn't needed. If you really want the parent to be called in all cases, just do if(can_be_cut) stuff stuff

File: code/modules/clothing/gloves/color.dm [hunk 4]
```diff
@@ -45,11 +46,12 @@
 
 /obj/item/clothing/gloves/color/black/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
 	if(istype(W, /obj/item/weapon/wirecutters))
-		if(icon_state == initial(icon_state)) //only if not dyed
-			user << "<span class='notice'>You snip the fingertips off of [src].</span>"
-			playsound(user.loc,'sound/items/Wirecutter.ogg', rand(10,50), 1)
-			new /obj/item/clothing/gloves/fingerless(user.loc)
-			qdel(src)
+		if(can_be_cut)
+			if(icon_state == initial(icon_state)) //only if not dyed
```
Comment (PKPenguin321) [inline code/modules/clothing/gloves/color.dm]: can this and the line above it be made into one if( && ) check

Comment (phil235) [inline code/modules/clothing/gloves/color.dm]: if you want :shrug:

**Pull request merged**